project-base/tests/App/Performance/Feed/AllFeedsTest.php

```diff
@@ -45,7 +45,7 @@ protected function setUp(): void
             'debug' => EnvironmentType::isDebug(EnvironmentType::TEST),
         ]);
 
-        $container = self::$kernel->getContainer();
+        $container = static::bootKernel()->getContainer();
```

Comment:
could this be done by injection?

Reply:
That would probably work, but in truth I do not want to refactor another part of the application (performance tests, unit tests, ...) that should have been modified in the past :) I will create a new US.